Utilities/crd-extractor.sh

```diff
@@ -50,10 +56,10 @@ NUM_OF_CRDS=0
 while read -r crd
 do
     filename=${crd%% *}
-    kubectl get crds "$filename" -o yaml > "$TMP_CRD_DIR/$filename.yaml" 2>&1
+    kubectl get crds "$filename" -o yaml > "$TMP_CRD_DIR/$filename.yaml" 2>&1q
```

Comment:
I'm curious. What's the trailing `q` for?

Reply:
Looks to me to be a typo!

Reply:
Yes, sorry for the delayed response. This is definitely a typo. I'll send out a fix today hopefully.

Reply:
I can make it very easy for you.

```suggestion
    kubectl get crds "$filename" -o yaml > "$TMP_CRD_DIR/$filename.yaml" 2>&1
```